[2024-10-01T21:53:55.711+0300] {executor_loader.py:254} INFO - Loaded executor: SequentialExecutor
[2024-10-01T21:53:55.730+0300] {scheduler_job_runner.py:935} INFO - Starting the scheduler
[2024-10-01T21:53:55.731+0300] {scheduler_job_runner.py:942} INFO - Processing each file at most -1 times
[2024-10-01T21:53:55.735+0300] {manager.py:174} INFO - Launched DagFileProcessorManager with pid: 971707
[2024-10-01T21:53:55.736+0300] {scheduler_job_runner.py:1847} INFO - Adopting or resetting orphaned tasks for active dag runs
[2024-10-01T21:53:55.737+0300] {settings.py:63} INFO - Configured default timezone UTC
[2024-10-01T21:53:55.748+0300] {manager.py:406} WARNING - Because we cannot use more than 1 thread (parsing_processes = 2) when using sqlite. So we set parallelism to 1.
[2024-10-01T21:54:15.438+0300] {dag.py:4180} INFO - Setting next_dagrun for Symptom2disease to 2024-10-01 18:50:00+00:00, run_after=2024-10-01 18:55:00+00:00
Dag run  in running state
Dag information Queued at: 2024-10-01 18:54:15.432065+00:00 hash info: 30ab03299000128b10991087036361f7
Dag run  in running state
Dag information Queued at: 2024-10-01 18:54:14.366302+00:00 hash info: 30ab03299000128b10991087036361f7
[2024-10-01T21:54:15.468+0300] {scheduler_job_runner.py:423} INFO - 2 tasks up for execution:
	<TaskInstance: Symptom2disease.preprocess_data scheduled__2024-10-01T18:45:00+00:00 [scheduled]>
	<TaskInstance: Symptom2disease.preprocess_data manual__2024-10-01T18:54:14.353435+00:00 [scheduled]>
[2024-10-01T21:54:15.468+0300] {scheduler_job_runner.py:495} INFO - DAG Symptom2disease has 0/16 running and queued tasks
[2024-10-01T21:54:15.468+0300] {scheduler_job_runner.py:495} INFO - DAG Symptom2disease has 1/16 running and queued tasks
[2024-10-01T21:54:15.468+0300] {scheduler_job_runner.py:634} INFO - Setting the following tasks to queued state:
	<TaskInstance: Symptom2disease.preprocess_data scheduled__2024-10-01T18:45:00+00:00 [scheduled]>
	<TaskInstance: Symptom2disease.preprocess_data manual__2024-10-01T18:54:14.353435+00:00 [scheduled]>
[2024-10-01T21:54:15.470+0300] {scheduler_job_runner.py:736} INFO - Trying to enqueue tasks: [<TaskInstance: Symptom2disease.preprocess_data scheduled__2024-10-01T18:45:00+00:00 [scheduled]>, <TaskInstance: Symptom2disease.preprocess_data manual__2024-10-01T18:54:14.353435+00:00 [scheduled]>] for executor: SequentialExecutor(parallelism=32)
[2024-10-01T21:54:15.470+0300] {scheduler_job_runner.py:680} INFO - Sending TaskInstanceKey(dag_id='Symptom2disease', task_id='preprocess_data', run_id='scheduled__2024-10-01T18:45:00+00:00', try_number=1, map_index=-1) to SequentialExecutor with priority 3 and queue default
[2024-10-01T21:54:15.470+0300] {base_executor.py:168} INFO - Adding to queue: ['airflow', 'tasks', 'run', 'Symptom2disease', 'preprocess_data', 'scheduled__2024-10-01T18:45:00+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:54:15.470+0300] {scheduler_job_runner.py:680} INFO - Sending TaskInstanceKey(dag_id='Symptom2disease', task_id='preprocess_data', run_id='manual__2024-10-01T18:54:14.353435+00:00', try_number=1, map_index=-1) to SequentialExecutor with priority 3 and queue default
[2024-10-01T21:54:15.470+0300] {base_executor.py:168} INFO - Adding to queue: ['airflow', 'tasks', 'run', 'Symptom2disease', 'preprocess_data', 'manual__2024-10-01T18:54:14.353435+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:54:15.472+0300] {sequential_executor.py:84} INFO - Executing command: ['airflow', 'tasks', 'run', 'Symptom2disease', 'preprocess_data', 'scheduled__2024-10-01T18:45:00+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:54:16.380+0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/sofia/Документы/Symptom2Disease/services/airflow/dags/pipeline_dag.py
[2024-10-01T21:54:16.466+0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sofia/Документы/Symptom2Disease/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-01T21:54:16.466+0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:54:16.472+0300] {example_python_operator.py:93} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:54:16.490+0300] {example_python_decorator.py:80} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:54:16.549+0300] {example_kubernetes_executor.py:39} WARNING - The example_kubernetes_executor example DAG requires the kubernetes provider. Please install it with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:54:16.557+0300] {tutorial_taskflow_api_virtualenv.py:29} WARNING - The tutorial_taskflow_api_virtualenv example DAG requires virtualenv, please install it.
[2024-10-01T21:54:16.577+0300] {task_command.py:467} INFO - Running <TaskInstance: Symptom2disease.preprocess_data scheduled__2024-10-01T18:45:00+00:00 [queued]> on host sofia-ASUS-TUF-Dash-F15-FX517ZC-FX517ZC
[2024-10-01T21:54:17.854+0300] {sequential_executor.py:84} INFO - Executing command: ['airflow', 'tasks', 'run', 'Symptom2disease', 'preprocess_data', 'manual__2024-10-01T18:54:14.353435+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:54:18.778+0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/sofia/Документы/Symptom2Disease/services/airflow/dags/pipeline_dag.py
[2024-10-01T21:54:18.860+0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sofia/Документы/Symptom2Disease/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-01T21:54:18.860+0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:54:18.866+0300] {example_python_operator.py:93} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:54:18.885+0300] {example_python_decorator.py:80} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:54:18.944+0300] {example_kubernetes_executor.py:39} WARNING - The example_kubernetes_executor example DAG requires the kubernetes provider. Please install it with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:54:18.951+0300] {tutorial_taskflow_api_virtualenv.py:29} WARNING - The tutorial_taskflow_api_virtualenv example DAG requires virtualenv, please install it.
[2024-10-01T21:54:18.966+0300] {task_command.py:467} INFO - Running <TaskInstance: Symptom2disease.preprocess_data manual__2024-10-01T18:54:14.353435+00:00 [queued]> on host sofia-ASUS-TUF-Dash-F15-FX517ZC-FX517ZC
[2024-10-01T21:54:20.117+0300] {scheduler_job_runner.py:764} INFO - Received executor event with state success for task instance TaskInstanceKey(dag_id='Symptom2disease', task_id='preprocess_data', run_id='scheduled__2024-10-01T18:45:00+00:00', try_number=1, map_index=-1)
[2024-10-01T21:54:20.118+0300] {scheduler_job_runner.py:764} INFO - Received executor event with state success for task instance TaskInstanceKey(dag_id='Symptom2disease', task_id='preprocess_data', run_id='manual__2024-10-01T18:54:14.353435+00:00', try_number=1, map_index=-1)
[2024-10-01T21:54:20.122+0300] {scheduler_job_runner.py:801} INFO - TaskInstance Finished: dag_id=Symptom2disease, task_id=preprocess_data, run_id=manual__2024-10-01T18:54:14.353435+00:00, map_index=-1, run_start_date=2024-10-01 18:54:18.990104+00:00, run_end_date=2024-10-01 18:54:19.833278+00:00, run_duration=0.843174, state=success, executor=SequentialExecutor(parallelism=32), executor_state=success, try_number=1, max_tries=1, job_id=87, pool=default_pool, queue=default, priority_weight=3, operator=BashOperator, queued_dttm=2024-10-01 18:54:15.469284+00:00, queued_by_job_id=85, pid=971946
[2024-10-01T21:54:20.122+0300] {scheduler_job_runner.py:801} INFO - TaskInstance Finished: dag_id=Symptom2disease, task_id=preprocess_data, run_id=scheduled__2024-10-01T18:45:00+00:00, map_index=-1, run_start_date=2024-10-01 18:54:16.598666+00:00, run_end_date=2024-10-01 18:54:17.515242+00:00, run_duration=0.916576, state=success, executor=SequentialExecutor(parallelism=32), executor_state=success, try_number=1, max_tries=1, job_id=86, pool=default_pool, queue=default, priority_weight=3, operator=BashOperator, queued_dttm=2024-10-01 18:54:15.469284+00:00, queued_by_job_id=85, pid=971894
[2024-10-01T21:54:20.268+0300] {scheduler_job_runner.py:423} INFO - 2 tasks up for execution:
	<TaskInstance: Symptom2disease.train_model scheduled__2024-10-01T18:45:00+00:00 [scheduled]>
	<TaskInstance: Symptom2disease.train_model manual__2024-10-01T18:54:14.353435+00:00 [scheduled]>
[2024-10-01T21:54:20.268+0300] {scheduler_job_runner.py:495} INFO - DAG Symptom2disease has 0/16 running and queued tasks
[2024-10-01T21:54:20.268+0300] {scheduler_job_runner.py:495} INFO - DAG Symptom2disease has 1/16 running and queued tasks
[2024-10-01T21:54:20.269+0300] {scheduler_job_runner.py:634} INFO - Setting the following tasks to queued state:
	<TaskInstance: Symptom2disease.train_model scheduled__2024-10-01T18:45:00+00:00 [scheduled]>
	<TaskInstance: Symptom2disease.train_model manual__2024-10-01T18:54:14.353435+00:00 [scheduled]>
[2024-10-01T21:54:20.269+0300] {scheduler_job_runner.py:736} INFO - Trying to enqueue tasks: [<TaskInstance: Symptom2disease.train_model scheduled__2024-10-01T18:45:00+00:00 [scheduled]>, <TaskInstance: Symptom2disease.train_model manual__2024-10-01T18:54:14.353435+00:00 [scheduled]>] for executor: SequentialExecutor(parallelism=32)
[2024-10-01T21:54:20.270+0300] {scheduler_job_runner.py:680} INFO - Sending TaskInstanceKey(dag_id='Symptom2disease', task_id='train_model', run_id='scheduled__2024-10-01T18:45:00+00:00', try_number=1, map_index=-1) to SequentialExecutor with priority 2 and queue default
[2024-10-01T21:54:20.270+0300] {base_executor.py:168} INFO - Adding to queue: ['airflow', 'tasks', 'run', 'Symptom2disease', 'train_model', 'scheduled__2024-10-01T18:45:00+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:54:20.270+0300] {scheduler_job_runner.py:680} INFO - Sending TaskInstanceKey(dag_id='Symptom2disease', task_id='train_model', run_id='manual__2024-10-01T18:54:14.353435+00:00', try_number=1, map_index=-1) to SequentialExecutor with priority 2 and queue default
[2024-10-01T21:54:20.270+0300] {base_executor.py:168} INFO - Adding to queue: ['airflow', 'tasks', 'run', 'Symptom2disease', 'train_model', 'manual__2024-10-01T18:54:14.353435+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:54:20.272+0300] {sequential_executor.py:84} INFO - Executing command: ['airflow', 'tasks', 'run', 'Symptom2disease', 'train_model', 'scheduled__2024-10-01T18:45:00+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:54:21.174+0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/sofia/Документы/Symptom2Disease/services/airflow/dags/pipeline_dag.py
[2024-10-01T21:54:21.254+0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sofia/Документы/Symptom2Disease/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-01T21:54:21.254+0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:54:21.260+0300] {example_python_operator.py:93} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:54:21.279+0300] {example_python_decorator.py:80} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:54:21.338+0300] {example_kubernetes_executor.py:39} WARNING - The example_kubernetes_executor example DAG requires the kubernetes provider. Please install it with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:54:21.345+0300] {tutorial_taskflow_api_virtualenv.py:29} WARNING - The tutorial_taskflow_api_virtualenv example DAG requires virtualenv, please install it.
[2024-10-01T21:54:21.361+0300] {task_command.py:467} INFO - Running <TaskInstance: Symptom2disease.train_model scheduled__2024-10-01T18:45:00+00:00 [queued]> on host sofia-ASUS-TUF-Dash-F15-FX517ZC-FX517ZC
[2024-10-01T21:55:28.398+0300] {sequential_executor.py:84} INFO - Executing command: ['airflow', 'tasks', 'run', 'Symptom2disease', 'train_model', 'manual__2024-10-01T18:54:14.353435+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:55:29.351+0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/sofia/Документы/Symptom2Disease/services/airflow/dags/pipeline_dag.py
[2024-10-01T21:55:29.435+0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sofia/Документы/Symptom2Disease/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-01T21:55:29.436+0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:55:29.442+0300] {example_python_operator.py:93} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:55:29.462+0300] {example_python_decorator.py:80} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:55:29.524+0300] {example_kubernetes_executor.py:39} WARNING - The example_kubernetes_executor example DAG requires the kubernetes provider. Please install it with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:55:29.532+0300] {tutorial_taskflow_api_virtualenv.py:29} WARNING - The tutorial_taskflow_api_virtualenv example DAG requires virtualenv, please install it.
[2024-10-01T21:55:29.547+0300] {task_command.py:467} INFO - Running <TaskInstance: Symptom2disease.train_model manual__2024-10-01T18:54:14.353435+00:00 [queued]> on host sofia-ASUS-TUF-Dash-F15-FX517ZC-FX517ZC
[2024-10-01T21:56:38.279+0300] {scheduler_job_runner.py:764} INFO - Received executor event with state success for task instance TaskInstanceKey(dag_id='Symptom2disease', task_id='train_model', run_id='scheduled__2024-10-01T18:45:00+00:00', try_number=1, map_index=-1)
[2024-10-01T21:56:38.280+0300] {scheduler_job_runner.py:764} INFO - Received executor event with state success for task instance TaskInstanceKey(dag_id='Symptom2disease', task_id='train_model', run_id='manual__2024-10-01T18:54:14.353435+00:00', try_number=1, map_index=-1)
[2024-10-01T21:56:38.282+0300] {scheduler_job_runner.py:801} INFO - TaskInstance Finished: dag_id=Symptom2disease, task_id=train_model, run_id=manual__2024-10-01T18:54:14.353435+00:00, map_index=-1, run_start_date=2024-10-01 18:55:29.572541+00:00, run_end_date=2024-10-01 18:56:37.980435+00:00, run_duration=68.407894, state=success, executor=SequentialExecutor(parallelism=32), executor_state=success, try_number=1, max_tries=1, job_id=89, pool=default_pool, queue=default, priority_weight=2, operator=BashOperator, queued_dttm=2024-10-01 18:54:20.269375+00:00, queued_by_job_id=85, pid=1075363
[2024-10-01T21:56:38.283+0300] {scheduler_job_runner.py:801} INFO - TaskInstance Finished: dag_id=Symptom2disease, task_id=train_model, run_id=scheduled__2024-10-01T18:45:00+00:00, map_index=-1, run_start_date=2024-10-01 18:54:21.386750+00:00, run_end_date=2024-10-01 18:55:28.123917+00:00, run_duration=66.737167, state=success, executor=SequentialExecutor(parallelism=32), executor_state=success, try_number=1, max_tries=1, job_id=88, pool=default_pool, queue=default, priority_weight=2, operator=BashOperator, queued_dttm=2024-10-01 18:54:20.269375+00:00, queued_by_job_id=85, pid=971998
[2024-10-01T21:56:38.293+0300] {manager.py:293} ERROR - DagFileProcessorManager (PID=971707) last sent a heartbeat 138.05 seconds ago! Restarting it
[2024-10-01T21:56:38.297+0300] {process_utils.py:132} INFO - Sending Signals.SIGTERM to group 971707. PIDs of all processes in the group: [971707]
[2024-10-01T21:56:38.297+0300] {process_utils.py:87} INFO - Sending the signal Signals.SIGTERM to group 971707
[2024-10-01T21:56:38.390+0300] {process_utils.py:80} INFO - Process psutil.Process(pid=971707, status='terminated', exitcode=0, started='21:53:55') (971707) terminated with exit code 0
[2024-10-01T21:56:38.392+0300] {manager.py:174} INFO - Launched DagFileProcessorManager with pid: 1178466
[2024-10-01T21:56:38.395+0300] {settings.py:63} INFO - Configured default timezone UTC
[2024-10-01T21:56:38.400+0300] {job.py:229} INFO - Heartbeat recovered after 138.27 seconds
[2024-10-01T21:56:38.406+0300] {manager.py:406} WARNING - Because we cannot use more than 1 thread (parsing_processes = 2) when using sqlite. So we set parallelism to 1.
[2024-10-01T21:56:38.551+0300] {dag.py:4180} INFO - Setting next_dagrun for Symptom2disease to 2024-10-01 18:55:00+00:00, run_after=2024-10-01 19:00:00+00:00
Dag run  in running state
Dag information Queued at: 2024-10-01 18:56:38.547970+00:00 hash info: 30ab03299000128b10991087036361f7
[2024-10-01T21:56:38.577+0300] {scheduler_job_runner.py:423} INFO - 3 tasks up for execution:
	<TaskInstance: Symptom2disease.preprocess_data scheduled__2024-10-01T18:50:00+00:00 [scheduled]>
	<TaskInstance: Symptom2disease.deploy_model scheduled__2024-10-01T18:45:00+00:00 [scheduled]>
	<TaskInstance: Symptom2disease.deploy_model manual__2024-10-01T18:54:14.353435+00:00 [scheduled]>
[2024-10-01T21:56:38.577+0300] {scheduler_job_runner.py:495} INFO - DAG Symptom2disease has 0/16 running and queued tasks
[2024-10-01T21:56:38.577+0300] {scheduler_job_runner.py:495} INFO - DAG Symptom2disease has 1/16 running and queued tasks
[2024-10-01T21:56:38.577+0300] {scheduler_job_runner.py:495} INFO - DAG Symptom2disease has 2/16 running and queued tasks
[2024-10-01T21:56:38.577+0300] {scheduler_job_runner.py:634} INFO - Setting the following tasks to queued state:
	<TaskInstance: Symptom2disease.preprocess_data scheduled__2024-10-01T18:50:00+00:00 [scheduled]>
	<TaskInstance: Symptom2disease.deploy_model scheduled__2024-10-01T18:45:00+00:00 [scheduled]>
	<TaskInstance: Symptom2disease.deploy_model manual__2024-10-01T18:54:14.353435+00:00 [scheduled]>
[2024-10-01T21:56:38.578+0300] {scheduler_job_runner.py:736} INFO - Trying to enqueue tasks: [<TaskInstance: Symptom2disease.preprocess_data scheduled__2024-10-01T18:50:00+00:00 [scheduled]>, <TaskInstance: Symptom2disease.deploy_model scheduled__2024-10-01T18:45:00+00:00 [scheduled]>, <TaskInstance: Symptom2disease.deploy_model manual__2024-10-01T18:54:14.353435+00:00 [scheduled]>] for executor: SequentialExecutor(parallelism=32)
[2024-10-01T21:56:38.579+0300] {scheduler_job_runner.py:680} INFO - Sending TaskInstanceKey(dag_id='Symptom2disease', task_id='preprocess_data', run_id='scheduled__2024-10-01T18:50:00+00:00', try_number=1, map_index=-1) to SequentialExecutor with priority 3 and queue default
[2024-10-01T21:56:38.579+0300] {base_executor.py:168} INFO - Adding to queue: ['airflow', 'tasks', 'run', 'Symptom2disease', 'preprocess_data', 'scheduled__2024-10-01T18:50:00+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:56:38.579+0300] {scheduler_job_runner.py:680} INFO - Sending TaskInstanceKey(dag_id='Symptom2disease', task_id='deploy_model', run_id='scheduled__2024-10-01T18:45:00+00:00', try_number=1, map_index=-1) to SequentialExecutor with priority 1 and queue default
[2024-10-01T21:56:38.579+0300] {base_executor.py:168} INFO - Adding to queue: ['airflow', 'tasks', 'run', 'Symptom2disease', 'deploy_model', 'scheduled__2024-10-01T18:45:00+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:56:38.579+0300] {scheduler_job_runner.py:680} INFO - Sending TaskInstanceKey(dag_id='Symptom2disease', task_id='deploy_model', run_id='manual__2024-10-01T18:54:14.353435+00:00', try_number=1, map_index=-1) to SequentialExecutor with priority 1 and queue default
[2024-10-01T21:56:38.579+0300] {base_executor.py:168} INFO - Adding to queue: ['airflow', 'tasks', 'run', 'Symptom2disease', 'deploy_model', 'manual__2024-10-01T18:54:14.353435+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:56:38.582+0300] {sequential_executor.py:84} INFO - Executing command: ['airflow', 'tasks', 'run', 'Symptom2disease', 'preprocess_data', 'scheduled__2024-10-01T18:50:00+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:56:39.493+0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/sofia/Документы/Symptom2Disease/services/airflow/dags/pipeline_dag.py
[2024-10-01T21:56:39.579+0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sofia/Документы/Symptom2Disease/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-01T21:56:39.579+0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:56:39.586+0300] {example_python_operator.py:93} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:56:39.605+0300] {example_python_decorator.py:80} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:56:39.670+0300] {example_kubernetes_executor.py:39} WARNING - The example_kubernetes_executor example DAG requires the kubernetes provider. Please install it with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:56:39.678+0300] {tutorial_taskflow_api_virtualenv.py:29} WARNING - The tutorial_taskflow_api_virtualenv example DAG requires virtualenv, please install it.
[2024-10-01T21:56:39.694+0300] {task_command.py:467} INFO - Running <TaskInstance: Symptom2disease.preprocess_data scheduled__2024-10-01T18:50:00+00:00 [queued]> on host sofia-ASUS-TUF-Dash-F15-FX517ZC-FX517ZC
[2024-10-01T21:56:40.892+0300] {sequential_executor.py:84} INFO - Executing command: ['airflow', 'tasks', 'run', 'Symptom2disease', 'deploy_model', 'scheduled__2024-10-01T18:45:00+00:00', '--local', '--subdir', 'DAGS_FOLDER/pipeline_dag.py']
[2024-10-01T21:56:41.822+0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/sofia/Документы/Symptom2Disease/services/airflow/dags/pipeline_dag.py
[2024-10-01T21:56:41.908+0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/sofia/Документы/Symptom2Disease/venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-10-01T21:56:41.908+0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:56:41.914+0300] {example_python_operator.py:93} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:56:41.934+0300] {example_python_decorator.py:80} WARNING - The virtalenv_python example task requires virtualenv, please install it.
[2024-10-01T21:56:41.999+0300] {example_kubernetes_executor.py:39} WARNING - The example_kubernetes_executor example DAG requires the kubernetes provider. Please install it with: pip install apache-airflow[cncf.kubernetes]
[2024-10-01T21:56:42.007+0300] {tutorial_taskflow_api_virtualenv.py:29} WARNING - The tutorial_taskflow_api_virtualenv example DAG requires virtualenv, please install it.
[2024-10-01T21:56:42.024+0300] {task_command.py:467} INFO - Running <TaskInstance: Symptom2disease.deploy_model scheduled__2024-10-01T18:45:00+00:00 [queued]> on host sofia-ASUS-TUF-Dash-F15-FX517ZC-FX517ZC
